spec/system/authorizations_spec.rb

```diff
@@ -74,8 +74,6 @@ def fill_in_authorization_form
 
         within ".authorizations-list" do
           expect(page).to have_content("El padró")
-          expect(page).not_to have_link("El padró")
-          expect(page).to have_content(I18n.localize(authorization.granted_at, format: :long, locale: :ca))
```

Comment:
Why was this changed? I think this should be kept 😕

Reply:
They were both failing and I did a little digging to see it was removed in this PR in decidim, https://github.com/decidim/decidim/pull/2311/files